drivers/timer/ite_it8xxx2_timer.c

```diff
@@ -0,0 +1,326 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Jyunlin Chen <jyunlin.chen@ite.com.tw>
+ */
+
+#include <zephyr.h>
+#include <sys/util.h>
+#include <drivers/timer/system_timer.h>
+#include <soc.h>
+#include <sys/printk.h>
+
+/**
+ * Macro Define
+ */
+#define EXT_CTL_B           (EC_Register_Base_Address + 0x1F10)
```

Comment:
Where does EC_Register_Base_Address come from? Can you add comments. I would think that this would a relative offset from the base address of the device as specified in the device tree